src/Select.js

```diff
@@ -1089,6 +1090,7 @@ const Select = createClass({
 			'is-pseudo-focused': this.state.isPseudoFocused,
 			'is-searchable': this.props.searchable,
 			'has-value': valueArray.length,
+			'Select-rtl': this.props.rtl,
```

Comment:
This is a nit but it should probably be `Select--rtl` since rtl is a modifier. Also it feels like it should belong at the top just below the other modifiers. Can you update?